Templates/WebHook-Generic/index.js

```diff
@@ -1,6 +1,16 @@
 ﻿module.exports = function (context, data) {
-    context.res = {};
     context.log('Webhook was triggered!');
-    context.res = { body: 'WebHook processed successfully! \n' + JSON.stringify(data, null, " ") };
+
+    // Check if we got first/last properties
+    if("first" in data && "last" in data) {
+        context.res = { body: { greeting: 'Hello ' + data.first + " " + data.last + "!"}};
```

Comment:
This would also be easier to read if it weren't all crammed on one line :) Rest looks good.